Cleaned up docs to focus on runtime. (#2345)
acorn-io/runtime: docs/docs/10-home.md

Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/runtime/discussions), or [Issues](https://github.com/acorn-io/runtime/issues)!
:::

:::note

> These are the docs for installing and running Acorn runtime on your own Kubernetes clusters. To learn how to use Acorn to package and deploy your applications, and the Acorn hosted platform please visit [docs.acorn.io](https://docs.acorn.io).

Reviewer: nit: feels like this flows a little bit better to me; take it or leave it though.

```suggestion
These are the docs for installing and running Acorn runtime on your own Kubernetes clusters. To learn about packaging and deploying your applications using Acorn, or for information about the Acorn hosted platform, please visit [docs.acorn.io](https://docs.acorn.io).
```
